package Exceptions;

import java.io.FileNotFoundException;

import Test.Test;

public class Exceptions {
    public static void main(String[] args) {
         try {
        int data = 7 / 0;
        } 
      
       catch (Exception ex) {
        System.out.printf("2"); 
       } 
        
    }}
        
    

   

